src/src/Controller/IterationController.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use PivotalTrackerV5\Client;
+
+class IterationController extends AbstractController
+{
+    /**
+     * @Route("/project/{project_id<\d+>}/iteration/{iteration_id<\d+>}", name="iteration_detail")
+     */
+    public function detail(int $project_id, int $iteration_id)
+    {
+        $pivotal_tracker_client = new Client($_ENV['PIVOTAL_TRACKER_API_KEY'], $project_id);
+        $project = $pivotal_tracker_client->getProject($project_id, ['fields' => 'name']);
+        $iteration = $pivotal_tracker_client->getIteration($iteration_id, ['fields' => 'number,team_strength,accepted_points,effective_points,velocity,start,finish,stories(id,name,story_type,estimate,current_state,url,owner_ids),points,accepted,created,analytics']);
```

Comment:
Just to make `fields` a little less cumbersome:

```suggestion
        $iteration_fields = [
            'number',
            'team_strength',
            'accepted_points',
            'effective_points',
            'velocity',
            'start',
            'finish',
            'stories(id,name,story_type,estimate,current_state,url,owner_ids)',
            'points',
            'accepted',
            'created',
            'analytics'
        ];
        
        $iteration = $pivotal_tracker_client->getIteration($iteration_id, [
            'fields' => implode(',', $iteration_fields)
        ]);
```